rpmlint/checks/BinariesCheck.py

```diff
@@ -141,6 +142,13 @@ def _check_shared_library(self, pkg, pkgfile_path, path):
                 self.output.add_info('W', pkg, 'unused-direct-shlib-dependency',
                                      path, dependency)
 
+    def _check_opt_library_dependency(self, pkg, pkgfile_path, path):
+        if not self.readelf_parser.is_archive:
+            for dependency in self.ldd_parser.dependencies:
```

Comment:
maybe:
```if any(d.startswith("/opt/") for d in self.ldd_parser.dependencies):
    self.output.add_info('E', pkg, 'linked-against-opt-library', path, dependency)
```

Reply:
Probably yes, it's just a cosmetic change.